ibis/mapd/client.py

```diff
@@ -368,15 +371,26 @@ def __init__(
 
         self.execution_type = execution_type
 
-        self.con = pymapd.connect(
-            uri=uri,
-            user=user,
-            password=password,
-            host=host,
-            port=port,
-            dbname=database,
-            protocol=protocol,
-        )
+        if session_id:
+            if self.version < pkg_resources.parse_version('0.12.0'):
+                raise Exception('Must have pympad > 0.12 to use session ID')
```

Comment:
Can you add an `Exception` subclass for this? E.g., `PyMapDVersionError`. There's also a typo in `pympad`.

Reply:
Sure thing.